models/encode.py

```diff
@@ -89,5 +89,19 @@ def _load_model(self):
         self.embedding_cls.load_model(self.model, self.model_path)
         return
 
-    def encode(self, text: str, pooling: str, **kwargs):
+    def encode_op(self, text: str, pooling: str, **kwargs):
```

Comment:
No need to create this method, add `return self.embedding_cls.encode(text, pooling, **kwargs)` inside the encode method only